### ISSUE TITLE
In breakpoints context menu: Missing `+` in shortcuts

### ISSUE THREAD

**Issue by gerdneuman** [devtools-html/debugger.html#7652]
See screenshot. This is on Windows 10 (not sure about Mac or Linux):

![image](https://user-images.githubusercontent.com/12525563/50682410-0b33aa80-100f-11e9-8a55-60954c18e211.png)

For instance, instead of `CtrlB` it should be `Ctrl+B`.

The `+` should be added like done in https://github.com/devtools-html/debugger.html/pull/7507

**Comment by bamanczak** [comment]
I created a Pull Request in devtools-core (devtools-modules) where the defect originates from. However at the moment we have code duplication as both devtools-core (devtools-modules) and debugger.html contain the formatKeyShortcut method. I could remove it as well.

I would suggest keeping formatKeyShortcut() just in the devtools-modules and reuse that one everywhere in debugger.html.
